src/EnergyPlus/Autosizing/Base.hh

```diff
@@ -98,7 +98,8 @@ enum class AutoSizingType
     WaterHeatingCoilUASizing = 20,
     ZoneCoolingLoadSizing = 26,
     ZoneHeatingLoadSizing = 27,
-    Unknown = 0
+    Unknown = 0,
+    Num = 35
```

Comment:
So in the autosizing folder, we have this official list of possible sizing calculation methods.  The first problem that I am solving with this PR is that a number of them are full-on duplicated in DataHVACGlobals.  As of right now there are definitely a bunch of things that will follow in subsequent commits, but I wanted to try to tell the story of what I am doing as I go.

Reply:
Is `Unknown` the same as `Invalid`, then it should be set to -1.